drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorWrapper.java

```diff
@@ -51,6 +51,12 @@
 
     void validate(Simulation simulation, Settings settings, RemoteCallback<?> callback);
 
+    void onRefreshedModelContent(SimulationRunResult testResult);
+
+    void populateAdditionalDocks(String identifier);
+
+    void resetAdditionalDocks();
```

Comment:
I think it could be better to have a generic `populateDocks` and `resetDocks` and move all the logic in the wrapper, wdyt?

Reply:
@danielezonca I renamed it `populateDocks` as suggested. I moved all the logic into `Wrapper` classes.